maxim/apis/maxim_apis.py

```
@@ -689,45 +695,209 @@ def get_folders(self) -> List[Folder]:
+            try:
+                file_data, mime_type, size = self._process_attachment(attachment)
+                
+                # Validate file size (e.g., 100MB limit)
+                # MAX_FILE_SIZE = 100 * 1024 * 1024  # 100MB
+                # if size > MAX_FILE_SIZE:
+                #     raise ValueError(f"File size {size} bytes exceeds maximum allowed size of {MAX_FILE_SIZE} bytes")
+
+                # Validate MIME type
+                if not mime_type or mime_type == 'application/octet-stream':
+                    # Try to infer MIME type from file extension if available
+                    if hasattr(attachment, 'name') and attachment.name:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)